modules/ROOT/pages/indexes/semantic-indexes/vector-indexes.adoc

```diff
@@ -424,6 +557,25 @@ Cosine similarity rebounds the trigonometric cosine.
 image::cosine_similarity_equation.svg["The cosine of vector v and vector u is defined as half of the quanity 1 plus the scalar product of v hat u hat, which equals half of the quantity 1 plus the scalar product of vector v vector u over the product of the l2-norm of vector v and the l2 norm ov vector u, which exists in the bounded set of real numbers between 0 inclusive and 1 inclusive."]
 In the above equation the trigonometric cosine is given by the scalar product of the two unit vectors.
 
+[[indexes-vector-provider-differences]]
+== Compatibility index provider differences
+The default and preferred vector index provider is `vector-2.0`.
+Previously created `vector-1.0` indexes will continue to function.
+New indexes can still be created with the `vector-1.0` index provider if specified, see xref:indexes-vector-create[].
+
+Supported index schema::
+A single-label, single-property index for nodes.
+
+Supported property value type::
+`LIST<FLOAT>`.
+
+Supported vector dimensionality::
+`INTEGER` between `1` and `2048` _inclusive_.
+
+Cosine similarity vector validity::
+* All vector components can be represented finitely in {ieee-754} single precision.
+* Its {l2-norm} is non-zero and can be represented finitely in {ieee-754} single precision.
```

Comment:
What does this table have to do with the differences between vector-1.0 and vector-2.0? Or should it have its own heading?

Reply:
If I have understood it correctly, those are the things `vector-1.0` follows that differs from what `vector-2.0` does (which is in the main parts with footnotes to this section)